### ISSUE TITLE
Figure out why windows PECL builds fail

### ISSUE THREAD

**Issue by TysonAndre** [runkit7/runkit7#188]
https://windows.php.net/downloads/pecl/releases/runkit7/2.0.2/logs/

Output of buildconf is this. The `configure` and `make` output is empty.

```
Rebuilding configure.js
Now run 'configure --help'
```

**Comment by TysonAndre** [comment]
My comment in https://marc.info/?l=pecl-dev&m=155562978906879&w=2

The configure output is looking better in 2.0.3, but DLLs aren't building (the output \
of "make" is also empty). I think it's because rmtools is trying to build "runkit" as \
a "static" extension (built into php.exe?) instead of a "shared" (DLL) extension, but \
I don't have rmtools or a windows environment set up to test this. (e.g. if I look at \
APCu's log files, it's shared)

- This probably needs to be changed from "--enable-runkit" to \
"--enable-runkit=shared" to build it as a DLL. data/config/pecl/exts.ini has \
                --enable-apc=shared (also has same "type=enable")
- this is in https://git.php.net/?p=web/rmtools.git;a=blobdiff;f=data/config/pecl/exts \
.ini;h=bdb9ff0b2f0bf757277a155abfb56bca5edac450;hp=5fe1176fe6f44764645867b7fb204510d2c \
1a7d5;hb=9bd1b4027ae837b0767558a737e81a4899d22155;hpb=0c6604b85ef978cfef60ebe4ac9b3f2037a1e5d2


> For the windows.php.net builds, I think they'd need to have the configure CLI \
> options changed from the default (not sure what) to `configure --enable-runkit`. \
> I'm not familiar with how this is changed.

My earlier suggestion(above) was assuming that runkit7 was outside of the php-src dir \
for rmtools builds, but I guess that's not the case.

The output of the configure script (I was a bit confused by why it called the option \
"--disable" before reading win32/build/confutils.js, but it just looks like it's \
setting runkit-super and runkit-modify to shared, which hopefully won't cause issues) \
(from https://windows.php.net/downloads/pecl/releases/runkit7/2.0.3/logs/)

> snapshot: forcing --disable-runkit-modify shared
> snapshot: forcing --disable-runkit-super shared
> snapshot: forcing runkit-spl_object_id on
> snapshot: forcing --disable-runkit-sandbox shared

and the part where it says it was built as a static extension:

```
Enabled extensions:
-------------------------
> Extension    | Mode   |
-------------------------
.....
> runkit       | static |
```

**Comment by TysonAndre** [comment]
I created a pr against the rmtools repo to avoid building this as `static`.

**Comment by TysonAndre** [comment]
This will be fixed in 3.0
